easybuild/easyconfigs/a/ASE/ASE-3.16.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'ASE'
+version = '3.16.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase/'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('Tkinter', '%(pyver)s', '-Python-%(pyver)s'),
+    ('matplotlib', '2.1.2', '-Python-%(pyver)s'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ase-build', 'bin/ase-db', 'bin/ase-gui', 'bin/ase-info', 'bin/ase-run'],
```

Comment:
and `bin/ase` itself?

Reply:
Copy paste mistake. I will fix this. I should probably drop at least ase-gui as well, as the newer versions tells you to use "ase gui" instead. Perhaps the same applies to all the other  commands?